client/modules/IDE/components/SketchList.jsx

```diff
@@ -9,13 +9,31 @@ import { bindActionCreators } from 'redux';
 import * as ProjectActions from '../actions/project';
 import * as SketchActions from '../actions/projects';
 import * as ToastActions from '../actions/toast';
+import * as IdeActions from '../actions/ide';
 
-const trashCan = require('../../../images/trash-can.svg');
+// const trashCan = require('../../../images/trash-can.svg');
```

Comment:
If the trashCan constant is no longer used. I'd suggest to remove this line of code entirely.